tests/serverpod_test_client/lib/src/http/http_status_html.dart

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Suggestion: Remove this definition.

Is it possible to reuse the definition from the client library instead?